package.json

```diff
@@ -123,6 +123,7 @@
   "dependencies": {
     "@thenewboston/ui": "^0.1.5",
     "@thenewboston/utils": "^1.0.0",
+    "app-root-path": "3.0.0",
```

Comment:
is `app-root-path` required? it seems like it is under dependency of `electron-window-state-manager`

<img width="704" alt="Screen Shot 2021-04-18 at 9 10 55 AM" src="https://user-images.githubusercontent.com/32864116/115130900-1109fa00-a026-11eb-8a64-716f34f27998.png">

Reply:
Glad it's working!. Yes app-root-path is needed by window-state-manager. Though it's installed as an own dependency i got errors and therefore added it to the main tree.